wiki/Contests/CCC/2024/en.md

```diff
@@ -0,0 +1,96 @@
+| Designer | ::{ flag=CN }:: [AlexDunk](https://osu.ppy.sh/users/9194799) |
+| Judge | ::{ flag=MY }:: [_Kobii](https://osu.ppy.sh/users/6209713), ::{ flag=SG }:: [Evening](https://osu.ppy.sh/users/2193881), ::{ flag=CN }:: [gzdongsheng](https://osu.ppy.sh/users/8660315), ::{ flag=ID }:: [Maxus](https://osu.ppy.sh/users/4335785), ::{ flag=CN }:: [Unpredictable](https://osu.ppy.sh/users/7560872) |
+| Statistician | ::{ flag=CN }:: [[Crz]Alleyne](https://osu.ppy.sh/users/11279273) |
+
+## Links
+
+- [Forum thread](https://osu.ppy.sh/community/forums/topics/1928815)
+- [Results Sheet](https://mappersguild.com/contests/results?contest=66026a8aa8c1495bcfbd23ea)
+- [Results Video](https://www.bilibili.com/video/BV1ZFv4enEiY/)
+- [Entries](https://drive.google.com/file/d/1IKg5NHqRxiqNuqAFSiq8rDMO-ADarGZX/view?usp=sharing)
+
+## Song List
+
+| Song | Length | BPM |
+| :-- | :-- | :-- |
+| you - Texture Garden | 02:28 | 128 |
+| Kirara Magic & Ulchero - Online Fantasy (feat. Moon Jelly) | 03:45 | 126 |
+| Essbee - Rebel Of Wildernes | 02:28 | 166 |
+| mybear - Last Slurp | 03:10 | 126 |
+| bobawhale & glitchless - starsignal! | 02:52 | 170 |
+| awfuless - Temptation | 03:04 | 173 |
+
+## Results
+
+| Place | Mapper(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") Winner | ::{ flag=CN }:: [BKwind](https://osu.ppy.sh/users/8900975) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") 2nd place | ::{ flag=CN }:: [\_Yiiiii](https://osu.ppy.sh/users/6066359) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") 3rd place | ::{ flag=CN }:: [tyrcs](https://osu.ppy.sh/users/13026904) |
+| Participation Award (4th-7th) | ::{ flag=CN }:: [Boomushroom](https://osu.ppy.sh/users/10588550), ::{ flag=CN }:: [Seiran-](https://osu.ppy.sh/users/14351534), ::{ flag=RU }:: [FlappiBakuse](https://osu.ppy.sh/users/11724295), ::{ flag=CN }:: [wIntEr7z](https://osu.ppy.sh/users/16509588) |
+
+## Ruleset
+
+### Contest rules
+
+- **This contest is for the `osu!mania` mode only.**
+- **This contest is for Chinese players only, including players with the flags of ::{ flag=CN }:: Chinese mainland, ::{ flag=HK }:: Hong Kong, ::{ flag=MO }:: Macao, and ::{ flag=TW }:: Taiwan eligible for participation.** If your flags aren't those four but actually it, you can participate this contest if player can prove that.
+- **Your account must be valid and can log in [Mappers' Guild](https://mappersguild.com).**
+- **Submissions must be in the `.osz` format.**
+- **The osu! client must be able to load and read your submission.**
+- **Submission must be `4K`, `5K`, `6K` or `7K`.** Other key modes are not considered in this contest (including n+1 playstyle).
+- **Beatmaps' mapping procress should be based on [Ranking Criteria.](/wiki/Ranking_criteria/osu!mania)** There are no requirements to follow RC compulsory, but the beatmap needs to be used Ranking Criteria as a fundamental reference, or that will effect your final score.
```

Comment:
```suggestion
- **Beatmap mapping process should be based on [Ranking Criteria](/wiki/Ranking_criteria/osu!mania).** While it is not compulsory to follow the Ranking Criteria strictly, the beatmap should use the Ranking Criteria as a fundamental reference, or this will affect the final score.

```